plugin/evm/atomic/state/atomic_trie_test.go

```diff
@@ -371,14 +359,16 @@ func TestAtomicTrieDoesNotSkipBonusBlocks(t *testing.T) {
 	assert.NotEqual(t, common.Hash{}, rootHash)
 
 	// Verify the operations are as expected
-	verifyOperations(t, atomicTrie, codec, rootHash, 1, expectedCommitHeight, operationsMap)
+	verifyOperations(t, atomicTrie, atomic.TestTxCodec, rootHash, 1, expectedCommitHeight, operationsMap)
 }
 
 func TestIndexingNilShouldNotImpactTrie(t *testing.T) {
 	// operations to index
 	ops := make([]map[ids.ID]*avalancheatomic.Requests, 0)
 	for i := 0; i <= testCommitInterval; i++ {
-		ops = append(ops, mustAtomicOps(atomic.GenerateTestImportTx()))
+		atomicOps, err := atomictest.ConvertToAtomicOps(atomic.GenerateTestImportTx())
+		assert.NoError(t, err)
```

Comment:
shouldn't we stop the test in case of error?

```suggestion
		require.NoError(t, err)
```

Reply:
I don't think `require` was around when these test were first written. We should probably do a huge change to all asserts in the codebase and replace them with require in a separate PR.